src/Symfony/Component/DependencyInjection/Loader/PhpFileLoader.php

```diff
@@ -55,12 +56,12 @@ public function load($resource, string $type = null)
         $this->container->fileExists($path);
 
         // the closure forbids access to the private scope in the included file
-        $load = \Closure::bind(function ($path) use ($container, $loader, $resource, $type) {
+        $load = \Closure::bind(function ($path, $env) use ($container, $loader, $resource, $type) {
             return include $path;
         }, $this, ProtectedPhpFileLoader::class);
 
         try {
-            $callback = $load($path);
+            $callback = $load($path, $this->env);
```

Comment:
Do we need to inject then $env? Or should we just do the PHP8 attributes?

Reply:
Since Symfony 4.0 we have the solution to use `config/packages/dev/acme.php`.

Reply:
True, but that's unrelated. Also, we already inject a few variables in the scope. Adding $env next to them just makes sense.